modules/claims_api/lib/claims_api/evidence_waiver_pdf/pdf.rb

```diff
@@ -52,7 +55,18 @@ def fill_pdf(response)
         page1_options(response),
         flatten: true
       )
+
       @page1_path = temp_path
     end
+
+    def save_pdf(path)
```

Comment:
Would a `FileUtils.cp(path, output_path)` or `File.open(output_path, 'w'){|of| of.puts(File.read(path))}` work too? Is `save_pdf` just copying from 1 temp file path to another?

Reply:
[64f895d](https://github.com/department-of-veterans-affairs/vets-api/pull/10938/commits/64f895d3400fb0e5945b42e1057af4d2317e0e8b)

Reply:
@acovrig better approach?